libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -472,10 +442,27 @@ export class ModelOverview extends React.Component<
             updateSelectedMetrics={this.onMetricConfigurationChange}
             selectableMetrics={selectableMetrics}
           />
-          {someCohortSelected && (
+          {((this.state.selectedFeatureBasedCohorts !== undefined &&
```

Comment:
shouldn't there be a check for `this.state.datasetCohortChartIsVisible` additionally?

```
{((!this.state.datasetCohortChartIsVisible && this.state.selectedFeatureBasedCohorts !== undefined &&
             this.state.selectedFeatureBasedCohorts.length > 0) ||
            (this.state.datasetCohortChartIsVisible &&
             this.state.selectedDatasetCohorts !== undefined &&
              this.state.selectedDatasetCohorts.length > 0)) && ( ...
```

Reply:
done